src/avram/callbacks.cr

```diff
@@ -80,6 +120,65 @@ module Avram::Callbacks
     end
   end
 
+  # Run the given method after `run` is called on an `Operation`.
+  # The return value of the `run` method is passed to `method_name`.
+  #
+  # ```
+  # after_run :log_entry
+  #
+  # private def log_entry(value)
+  #   log_stuff(value)
+  # end
+  # ```
+  macro after_run(method_name)
+    def after_run(object)
+      {% if @type.methods.map(&.name).includes?(:after_run.id) %}
+        previous_def
+      {% else %}
+        super
+      {% end %}
+
+      {{ method_name.id }}(object)
+    end
+  end
```

Comment:
This can delegate to the other macro like
```suggestion
  macro after_run(method_name)
    after_run { |object| {{ method_name.id }}(object) }
  end
```